pkcs8/src/common.rs

```diff
@@ -0,0 +1,76 @@
+use core::convert::TryFrom;
+
+use der::{Encodable, Encoder, Tagged};
+
+/// Version marker for PKCS#8 documents.
+///
+/// (RFC 5958 designates `0` and `1` as the only valid version for PKCS#8 documents.)
+#[derive(Clone, Debug, Copy)]
+pub enum Version {
+    /// Denotes PKCS#8 v1, used for [`crate::PrivateKeyInfo`] and [`crate::OneAsymmetricKey`]
+    V1 = 0,
+    /// Denotes PKCS#8 v2, only used for [`crate::OneAsymmetricKey`]
+    V2 = 1,
+}
+
+impl Into<u8> for Version {
```

Comment:
Why not `impl From<Version> for u8`? There's generally no advantage to implementing `Into` when implementing `From` is possible as well.